README.md

```diff
@@ -1,27 +1,32 @@
-<h> A/B Testing/h>
+# A/B Testing
 
 <ins>**Overview**</ins>
+
```

Comment:
@Iriakuma please look at the comments I left on your previous pr (before you closed), everything still applies:
https://github.com/Iriakuma/ABTest/pull/1

Reply:
I have edited, but it only reflects in the branch; the main still has all those errors. Do I delete the main and create it again, its not so clear to me

Reply:
Definitely do not do anything with main.
It should only reflect in the branch until it is merged.
Your branch still contains the bugs, so fix it in your branch first. Then we review and merge into main.